data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/extension/databasedownload/DBSourceOptions.java

```diff
@@ -17,7 +17,8 @@
 public enum DBSourceOptions {
     PATH("path"),
     URL("url"),
-    S3("s3");
+    S3("s3"),
+    CDN("cdn");
```

Comment:
Generically for the user, this should be configured as an HTTP endpoint.

I think you can make this:

```
HTTP("http")
```

The fact that the implementation is a CDN is not relevant for the user. That is a detail of the server itself, not the client (ie. Data Prepper).